assets/airports/rjss.json

```diff
@@ -0,0 +1,960 @@
+            "rwy": {
+                "RJSS27": ["#067"]
+            },
+            "draw": [
+                ["OWLET*", "DATTE", "RIBON", "SNOOK"]
+            ]
+        },
+        "OWLETW": {
+            "icao": "OWLETW",
+            "name": "Owlet West",
+            "entryPoints": {
+                "OWLET": [["OWLET", "A130+"]]
+            },
+            "body": [["PRINK", "A130+"], ["QUIST", "S250-"], ["RIBON", "A60+"]],
+            "rwy": {
+                "RJSS09": [["SHIPS", "A40+"]]
+            },
+            "draw": [
+                ["OWLET*", "PRINK", "QUIST", "RIBON", "SHIPS"]
+            ]
+        },
+        "PERID": {
+            "icao": "PERID",
+            "name": "Perid",
+            "entryPoints": {
+                "AOBAR": ["AOBAR"]
+            },
+            "body": ["_SDE135010", "_SDE120010", "_SDE105010"],
+            "rwy": {
+                "RJSS27": ["PERID"]
+            },
+            "draw": [
+                ["AOBAR*", "_SDE135010", "_SDE120010", "_SDE105010", "PERID"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "RJSS",
+            "destination": "",
+            "category": "departure",
+            "route": "RJSS27.STEED3.RIKYU",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 1.58,
+            "airlines": [
+                ["ana/short", 11],
+                ["ibx", 5],
+                ["jal/short", 8],
+                ["apj", 3],
+                ["sky", 2],
+                ["waj", 1]
+            ]
+        },
+        {
+            "origin": "RJSS",
+            "destination": "",
+            "category": "departure",
+            "route": "RJSS27.DERBY3.GTC",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.79,
+            "airlines": [
+                ["aar", 1],
+                ["ana/short", 1],
+                ["apj", 1],
+                ["cca", 1],
+                ["eva", 1],
+                ["fda", 1],
+                ["ibx", 7],
+                ["tha", 1],
+                ["ttw", 1],
+                ["jal/short", 2]
+            ]
+        },
+        {
+            "origin": "RJSS",
+            "destination": "",
+            "category": "departure",
+            "route": "RJSS27.CUBIC3.SAMBO",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.79,
+            "airlines": [
+                ["ado", 2],
+                ["ana/short", 3],
+                ["ibx", 2],
+                ["jal/short", 5],
+                ["apj", 3]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "RJSS",
+            "category": "arrival",
+            "route": "AIZUH.Y515.LANCE.LANCEA.RJSS27",
+            "altitude": [11000, 40000],
```

Comment:
Due to the fact that none of the arrival procedures contain at or below altitude restrictions, aircraft won't actually descend on the arrival. So any aircraft that spawn above FL160 will never enter the airspace. Only solution for this at the moment is to lower the upper bound for the spawn altitude. Same applies to other arrivals